### ISSUE TITLE
issue with interest HW.py

### ISSUE THREAD

**Issue by KewTeacher** [850078897/Chapter-2-4#2]
In interest HW.py you've combined the interest exercise here with the next exercise. Please remove the extra code

**Comment by 850078897** [comment]
p = 10000
n = 12
r = 0.08
t = int(input("the number of years: "))
final_amount = p * ((1 + r / n ) ** (n * t))
print (final_amount)

**Comment by KewTeacher** [comment]
that's right, but I wanted you to revise the file. It should be in two separate files. Please make one file for interest.py and one for alarm.py

Otherwise good work.
